docs/src/main/paradox/loglevels.md

```diff
@@ -40,3 +40,19 @@ For example using curl:
 ```
 curl -X PUT "http://127.0.0.1:8558/loglevel?logger=com.example.MyActor&level=DEBUG"
 ```
+
+#### Classic Akka Logger Level
+
+When using classic Akka and logging through the built in API there is an [additional level of filtering](https://doc.akka.io/docs/akka/current/logging.html#slf4j) using the
+`akka.loglevel` setting. If you have not set `akka.loglevel` to `DEBUG` (recommended) log entries from the classic logging API may never reach the logger backend att all.
+
+The current level configured with `akka.loglevel` can be inspected with a GET request to `akka/classic/loglevel`.
+
+If management `read-only` is set to `false` PUT requests to `akka/classic/loglevel?level=[level name]` will dynamically change that.
```

Comment:
I'm not fully convinced that we should call this Classic, since it's also used for internal logging (also when using Typed): https://doc.akka.io/docs/akka/current/typed/logging.html#internal-logging-by-akka

Reply:
Ah, good point. So perhaps just skip classic from that path then?

Reply:
update path